packages/core/src/Stores/common-store.js

```diff
@@ -280,13 +281,15 @@ export default class CommonStore extends BaseStore {
     setWithdrawURL(withdraw_url) {
         this.withdraw_url = withdraw_url;
     }
-
-    setServicesError(error) {
+    resetServicesError() {
+        this.services_error = {};
+    }
+    setServicesError(error, is_v2) {
```

Comment:
```suggestion
    setServicesError(error, hide_toast = false) {
```
v2 should not have any relationship with the core. Can we rename it to hide_toast?